backend/tests/sockets/twisted_sockets/test_smartsocket.py

```
@@ -466,22 +465,24 @@ def test_on_message_callback(binary_data_io: tuple[bytes, dict]):
-            "exchange": "NSE_CM",
+            "symbol": "symbol"
+            # "socket_name": "smartapi",
+            # "exchange": "NSE_CM",
         }
     )
     actual_call_args = json.loads(smartsocket.on_data_save_callback.call_args.args[0])
     assert actual_call_args.pop("retrieval_timestamp")
 
     assert smartsocket.on_data_save_callback.call_count == 1
-    assert actual_call_args == exptected_data
+    for keys in exptected_data.keys():
```

Ruff (0.8.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)